ibis/expr/api.py

```diff
@@ -1059,9 +1059,14 @@ def quantile(arg, quantile, interpolation='linear'):
 
     Returns
     -------
-    quantile : scalar type, same as input
+    quantile
+        if scalar input, scalar type, same as input
+        if array input, array type, same as input
     """
-    op = _ops.Quantile(arg, quantile, interpolation)
+    try:
+        op = _ops.Quantile(arg, quantile, interpolation)
+    except _com.IbisTypeError:
```

Comment:
I know I originally suggested this, but I'm not actually sure whether `IbisTypeError` can be thrown for any other reason here than `quantile` being the wrong type.

Probably better to just check `isinstance(quantile, collections.Sequence)`. I think it's more readable that way, and expresses the type checking intent more clearly.